data/generate_test_data.py

```
@@ -0,0 +1,294 @@
+    ]
+    
+    all_issues.extend(noise_issues)
+    
+    # Shuffle to mix them up
+    random.shuffle(all_issues)
+
+    print(f"Generated {len(all_issues)} issues to create.")
+
+    for i, issue in enumerate(all_issues):
+        print(f"Creating issue {i+1}/{len(all_issues)}: {issue['title']}")
+        response = requests.post(url, json=issue, headers=headers)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)